Sources/PointFree/Subscribe.swift

```diff
@@ -116,7 +114,23 @@ private func subscribe(
           "requiresAction": true,
         ]
       )
+    } else if [.incomplete, .incompleteExpired].contains(stripeSubscription.status) {
+      Task {
```

Comment:
could do `@Dependency(\.fireAndForget)` here if we wanna get in the habit of doing things like that.

Reply:
Coo coo, then let's skip!